test/test_cuda.py

```diff
@@ -2052,6 +2052,13 @@ def test_bincount_cuda(self):
         self.assertEqual(t.cpu().bincount(), t.bincount())
         self.assertEqual(t.cpu().bincount(w_cpu), t.bincount(w))
 
+    @skipIfRocm
+    def test_histc_cuda(self):
+        x = torch.tensor((2, 4, 2, 2, 5, 4)).to('cuda')
+        y = x.histc()
+        z = torch.tensor((0, 3, 0, 2, 1)).to('cuda')
+        self.assertEqual(z, y)
```

Comment:
test failed by,  `RuntimeError: _th_histc is not implemented for type torch.cuda.LongTensor`, it looks like it's not calling the cuda implementation but the cpu one, is that right?